src/rpc/net.cpp

```diff
@@ -272,6 +272,61 @@ static RPCHelpMan getpeerinfo()
     };
 }
 
+static RPCHelpMan addpermissionflags()
+{
+    return RPCHelpMan{"addpermissionflags",
+                "\nAdd permission flags to the peers using the given IP address (e.g. 1.2.3.4) or\n"
+                "CIDR-notated network (e.g. 1.2.3.0/24). Uses the same permissions as -whitebind.\n"
+                "Additional flags \"in\" and \"out\" control whether permissions apply to incoming and/or outgoing connections (default: incoming only).\n",
+                {
+                    {"flags", RPCArg::Type::ARR, RPCArg::Default{UniValue::VARR}, "The permission flags",
+                        {
+                            {"flag", RPCArg::Type::STR, RPCArg::Optional::OMITTED, "Permission flag"},
+                        }
+                    },
+                    {"ip|network", RPCArg::Type::STR, RPCArg::Optional::NO, "The IP address or network"}
+                },
+                RPCResult{RPCResult::Type::NONE, "", ""},
+                RPCExamples{
+                    HelpExampleCli("addpermissionflags", "[\"noban\", \"mempool\", \"in\", \"out\"] \"127.0.0.1\"")
+            + HelpExampleRpc("addpermissionflags", "[\"noban\", \"mempool\", \"in\", \"out\"], \"127.0.0.1\" }")
```

Comment:
Nit, probably only fix if file is touched anyway: one closing bracket "}" too much near the end of the line.